desktop_version/src/Editor.cpp

```diff
@@ -412,6 +412,8 @@ void editorclass::reset(void)
 
     state = EditorState_DRAW;
     substate = EditorSubState_MAIN;
+
+    undo_buffer.clear();
```

Comment:
When a new level is loaded, the undo buffer is cleared but not the redo buffer. So if there's anything in the redo buffer, you can press Ctrl+Y and that will be applied in the new level when it shouldn't be.